apple/DemoApp/Demo/DemoNavigationView.swift

```diff
@@ -128,6 +128,8 @@ struct DemoNavigationView: View {
                                     }
                                 } label: {
                                     Text("Start Nav")
+                                        .lineLimit(1)
+                                        .minimumScaleFactor(0.5)
```

Comment:
This is unrelated, but I noticed on small devices with slightly scaled up text:

<img width="359" alt="Screenshot 2024-09-18 at 21 18 25" src="https://github.com/user-attachments/assets/44716e19-3237-4c0b-8688-b3ed2d4e59f7">